tpgtools/property.go

```diff
@@ -868,6 +890,16 @@ func createPropertiesFromSchema(schema *openapi.Schema, typeFetcher *TypeFetcher
 		props = append(props, p)
 	}
 
+	// handle conflict fields
+	for i, _ := range props {
+		p := &props[i]
+		if p.JSONCaseConflictsWith != nil {
+			for _, cf := range p.JSONCaseConflictsWith {
+				p.ConflictsWith = append(p.ConflictsWith, conflictsMap[cf])
```

Comment:
Are we guaranteed to have an entry in the conflictsMap for every value of cf?

Reply:
Will add a check here and throw an error if we don't get an entry for some value of cf, as I think there should be one for each cf if it got generate and covert correctly.